src/routes/+page.svelte

```diff
@@ -195,6 +271,49 @@
         max-width: 100%;
     }
 
+    .welcome :global(.scroller),
+    .welcome :global(.message) {
+        overflow: visible;
+    }
+
+    .welcome :global(.message) {
+        line-height: 1.7;
+    }
+
+    .locale-button-wrapper {
+        display: inline-block;
+    }
+
+    :global(button.locale-picker.background.padding) {
```

Comment:
This feels a bit hacky, and the tighter spacing for the buttons looks reasonable. I recommend removing this and just changing the padding for all buttons with backgrounds in Button.svelte.